pages/common/clido.md

```diff
@@ -0,0 +1,36 @@
+# clido
+
+> Save-state TODO app for the terminal.
+> For more info, visit the Clido Wiki: <https://codeberg.org/Oglo12/clido/wiki>.
+
+- Create a list:
+
+`clido --new {{name}}`
+
+- Load a list:
+
+`clido --load {{name}}`
+
+- Delete a list:
+
+`clido --remove {{name}}`
+
+- List all lists:
+
+`clido --lists`
+
+- Toggle autowrite:
+
+`clido toggle-autowrite`
+
+- Open a list in a text editor:
+
+`clido edit {{texteditor}}`
```

Comment:
`snake_case` should be used for multi-word placeholders.
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming
```suggestion
`clido edit {{text_editor}}`
```